src/views/About/index.jsx

```
@@ -1,7 +1,45 @@
+  return (
+    <div className="p-6 md:p-12 bg-white font-sans text-gray-700 max-w-4xl mx-auto">
+      <h1 className="text-3xl font-bold text-black mb-6 border-b pb-2">
+        About Us
+      </h1>
+      <p className="text-lg leading-relaxed mb-4">
+        <span className="font-semibold text-black">WotchGotcha</span> is a
+        dynamic digital platform built to empower creativity, self-expression,
+        and community storytelling.
+      </p>
+      <p className="text-lg leading-relaxed mb-4">
+        Whether it's sharing a compelling video in{" "}
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)